nncf/tensorflow/pruning/tensor_processor.py

```diff
@@ -50,3 +50,10 @@ def repeat(cls, tensor: NNCFTensor, repeats: int) -> NNCFTensor:
     def elementwise_mask_propagation(cls, input_masks: List[NNCFTensor]) -> NNCFTensor:
         cls.assert_allclose(input_masks)
         return input_masks[0]
+
+    @classmethod
+    def split(cls, tensor: NNCFTensor, output_shapes: List[int]) -> List[NNCFTensor]:
+        # pylint: disable=E1120
```

Comment:
Just curious why the same function for PT does not lead to any pylint issue, but TF one needs that exception
https://github.com/openvinotoolkit/nncf/pull/1244/files#diff-a0b1a6b755a3b20734a5dba200a6c1a47f11e6d6fb54e12376971f586a24fd21R52-R56

Reply:
E1120 is [no value for parameter](https://pylint.pycqa.org/en/latest/user_guide/messages/error/no-value-for-parameter.html), I actually fail to see where such a warning could be spawned in this code since it is a function declaration in this line, not a function call.

Reply:
It could be pylint false positive, which appear due to c++ source code binding. There was such cases before